src/worker/storage.ts

```diff
@@ -42,8 +51,18 @@ export class PredictiveTextStudioDexie extends Dexie {
        * +------------------+
        */
       files: "++id, name, wordlist",
+      /**
+       * packageInfo Table Scehma
```

Comment:
What does `packageInfo` store? What is the intended purpose of what we store here? What might go in here in the future?